platform/qt/include/qmapbox.hpp

```diff
@@ -26,6 +26,12 @@ struct Q_DECL_EXPORT Feature {
         LineStringType,
         PolygonType
     };
+
+    Feature() {}
+
+    Feature(Type type_, const CoordinatesCollections& geometry_, const QVariantMap& properties_, const QVariant& id_)
+        : type(type_), geometry(geometry_), properties(properties_), id(id_) {}
```

Comment:
Let's use default values for this ctor to avoid having two ctors - we've decided on this approach when implementing support for GCC 4.9 elsewhere in GL native code.